tests/VCR/Storage/AbstractStorageTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace VCR\Storage;
+
+use org\bovigo\vfs\vfsStream;
+
+/**
+ * Test integration of PHPVCR with PHPUnit.
+ */
+class AbstractStorageTest extends \PHPUnit_Framework_TestCase
+{
+    protected $handle;
+    protected $filePath;
+    protected $storage;
+
+    public function testFilePathCreated()
+    {
+        $fs = vfsStream::setup('test');
+
+        $this->storage = new TestStorage(vfsStream::url('test/'), 'file');
+        $this->assertTrue($fs->hasChild('file'));
+
+        $this->storage = new TestStorage(vfsStream::url('test/'), 'folder/file');
```

Comment:
ah. great. somebody claimed that i can just pass 0 to mkdir for permission mask, as that would trigger the default. but that seems to be wrong and the doc does not claim it would work. using 0777 things work now.